mod-tool/src/main/scala/at/forsyte/apalache/tla/Tool.scala

```diff
@@ -354,22 +354,48 @@ object Tool extends App with LazyLogging {
   private def configure(config: ConfigCmd): Unit = {
     config.submitStats match {
       case Some(isEnabled) =>
-        val statCollector = new ExecutionStatisticsCollector()
-        if (isEnabled) {
-          logger.info("Statistics collection is ON.")
-          logger.info("This also enabled TLC and TLA+ Toolbox statistics.")
-          statCollector.set(Selection.ON)
-        } else {
-          logger.info("Statistics collection is OFF.")
-          logger.info("This also disabled TLC and TLA+ Toolbox statistics.")
-          statCollector.set(Selection.NO_ESC)
+        if (configDirExistsOrCreated()) {
+          val statCollector = new ExecutionStatisticsCollector()
+          // protect against potential exceptions in the tla2tools code
+          try {
+            if (isEnabled) {
+              statCollector.set(Selection.ON)
+              logger.info("Statistics collection is ON.")
+              logger.info("This also enabled TLC and TLA+ Toolbox statistics.")
+            } else {
+              statCollector.set(Selection.NO_ESC)
+              logger.info("Statistics collection is OFF.")
+              logger.info("This also disabled TLC and TLA+ Toolbox statistics.")
+            }
+          } catch {
+            case e: Exception =>
+              logger.warn(e.getMessage)
+              logger.warn(s"Unable to update statistics configuration. The other features will keep working.")
+          }
         }
```

Comment:
I suggest adding an else branch here to report to the user when the stats cannot be enabled (as a result of the dir creationg/check failing).

Reply:
Just did that.